llvm/test/CodeGen/AMDGPU/propagate-waves-per-eu.ll

```diff
@@ -279,7 +268,7 @@ define amdgpu_kernel void @kernel_3_6() #12 {
 ; 3,6 -> 6,9
 define internal void @refine_upper_func_3_6() #13 {
 ; CHECK-LABEL: define internal void @refine_upper_func_3_6
-; CHECK-SAME: () #[[ATTR9]] {
+; CHECK-SAME: () #[[ATTR14:[0-9]+]] {
```

Comment:
This is the last "real" change in the file. The waves/EU range for this function goes from [4,10] to [3,6]. All attribute changes below are just renames.